src/main/java/me/crespel/runtastic/RuntasticExportConverter.java

```diff
@@ -61,7 +69,25 @@ protected void doList(File path) throws FileNotFoundException, IOException {
 		SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
 		List<SportSession> sessions = converter.listSportSessions(path);
 		for (SportSession session : sessions) {
-			System.out.println(sdf.format(session.getStartTime()) + " - ID: " + session.getId() + ", type: " + session.getSportTypeId() + ", duration: " + session.getDuration() + "s");
+			System.out.println(sdf.format(session.getStartTime()) + " - ID: " + session.getId() + ", Sport Type: " + session.getSportTypeId() + ", duration: " + session.getDuration()/60000 + "min");

```

Comment:
Possible improvement: display the duration as hours:minutes:seconds for easier reading.

Reply:
Would be useful and nice. I don't know an "easy" fix.

Reply:
Found solution, use new Time(session.getDuration()).toString()